### PR TITLE
[FX-4629] Update Product Docs with Partial Scans: Reduced Scopes

### PR DESCRIPTION
**Pull request by mtsuji-cobalt**
### Added

- New dedicated page for Partial Scans: Reduced Scope

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Scans/reduced-scope.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+linkTitle: "Partial Scans: Reduced Scope"
+weight: 50
+description: Define a specific set of URLs within your target to scan
+---
+
+Rather than scanning the entire target, specify a subset of URLs within your target to scan by defining a Reduced Scope in your target settings. This is particularly useful when you want to focus on specific areas of your application, such as new features or high-risk areas.
+
+### Benefits
+
+- **Faster Feedback:** Get faster feedback on the security of your code changes. This is especially helpful in CI/CD pipelines, where you need to deliver code changes quickly and reliably.
```
Comment (TaylorBriggs) [inline content/en/Platform Deep Dive/Scans/reduced-scope.md:12]: Does the public API support this already? If not, we might not want to mention CI/CD yet.

**Review by TaylorBriggs — APPROVED** [review]

File: content/en/Platform Deep Dive/Scans/reduced-scope.md (hunk 1 above)
Comment (mahmoudadly-cobalt) [reply content/en/Platform Deep Dive/Scans/reduced-scope.md:12]: Yes, they can use this endpoint to trigger a new scan https://cobalt-public-api.netlify.app/v2/#create-a-dast-scan

File: content/en/Platform Deep Dive/Scans/reduced-scope.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Partial Scans: Reduced Scope"
+linkTitle: "Partial Scans: Reduced Scope"
+weight: 50
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/reduced-scope.md:4]: The integrations page has the same weight. Now this new page will be under it. Is that the desired order?

File: content/en/Platform Deep Dive/Scans/reduced-scope.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Go to your target's settings.
+2. In the Scanner tab, locate the **Reduced Scope** section.
+3. Add the URLs you want to include in your partial scan.
+   - Use wildcards (`*`) to include all sub-pages of a specific URL. For example, `https://www.example.com/blog/*` would include all pages within the blog section.
+4. Add and enable as many URLs as you need to define the intended reduced scope.
+   {{% image src="/deepdive/scans/partial-scans-configured.png" alt="Configured Reduced Scope" %}}
+6. Save your changes.
+7. Select **Scan Now** for the target to start a Partial Scan.
+{{% image src="/deepdive/scans/partial-scans-modal.png" alt="Reduced Scope Scan Now" %}}
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/reduced-scope.md:27]: The two modals look huge on the page. I suggest using a smaller screen width when taking screenshots in general, and especially for modals.

File: content/en/Platform Deep Dive/Scans/reduced-scope.md (hunk 2 above)
Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Scans/reduced-scope.md:4]: @mahmoudadly-cobalt i forgot to mention the ordering of the child pages - i think it should definitely be above Integrations (though not sure exactly where - maybe below Sequence Recorder or after Extra Hosts?) but i wasn't sure how to reorder it

**Review by mohitkyadav — APPROVED** [review]

File: content/en/Platform Deep Dive/Scans/reduced-scope.md (hunk 2 above)
Comment (mahmoudadly-cobalt) [reply content/en/Platform Deep Dive/Scans/reduced-scope.md]: I updated the order now.

**Review by mahmoudadly-cobalt — APPROVED** [review]

**Comment by mtsuji-cobalt** [comment]
@mahmoudadly-cobalt i tried to make the images smaller but i don't think it made a difference...

**Comment by mahmoudadly-cobalt** [comment]
@mtsuji-cobalt I refreshed now and they looked smaller than before.

**Pull request merged**